modules/common/src/pipes/constants/index.ts

```diff
@@ -0,0 +1,37 @@
+export enum ErentoLocales {
+    deDe = 'de-DE',
+    deCh = 'de-CH',
+    deAt = 'de-AT',
+    enGb = 'en-GB',
+    enUs = 'en-US',
```

Comment:
I'd like to propose a different property key naming here:
1. Star with uppercase (that is the best practise)
2. Name the key only after "terrritory", so US, GB etc.

The first point also applies to the Languages enums.

Reply:
problem with naming keys after territories is if we would have e.g. Canada specific content we would still need `enCa`and `frCa` as single territory can have multiple languages